commandapi-platforms/commandapi-bukkit/commandapi-bukkit-plugin-mojang-mapped/src/main/java/dev/jorel/commandapi/CommandAPIMain.java

```diff
@@ -51,7 +51,8 @@ public void onLoad() {
 			.missingExecutorImplementationMessage(fileConfig.getString("messages.missing-executor-implementation"))
 			.dispatcherFile(fileConfig.getBoolean("create-dispatcher-json") ? new File(getDataFolder(), "command_registration.json") : null)
 			.shouldHookPaperReload(fileConfig.getBoolean("hook-paper-reload"))
-			.skipReloadDatapacks(fileConfig.getBoolean("skip-initial-datapack-reload"));
+			.skipReloadDatapacks(fileConfig.getBoolean("skip-initial-datapack-reload"))
+			.beLenientForMinorVersions(fileConfig.getBoolean("lenient-for-minor-versions"));
```

Comment:
The name of this option in the `config.yml` is now `be-lenient-for-minor-versions` (Same in `commandapi-bukkit-plugin`)
```suggestion
			.beLenientForMinorVersions(fileConfig.getBoolean("be-lenient-for-minor-versions"));
```